internal/server/my_details_test.go

```diff
@@ -16,17 +16,28 @@ import (
 type mockMyDetailsClient struct {
 	count       int
 	lastCookies []*http.Cookie
+	lastRequest string
 	err         error
 	data        sirius.MyDetails
+	errors      sirius.ValidationErrors
 }
 
 func (m *mockMyDetailsClient) MyDetails(ctx context.Context, cookies []*http.Cookie) (sirius.MyDetails, error) {
 	m.count += 1
 	m.lastCookies = cookies
+	m.lastRequest = "MyDetails"
 
 	return m.data, m.err
 }
 
+func (m *mockMyDetailsClient) EditMyDetails(ctx context.Context, cookies []*http.Cookie, id int, phoneNumber string) (sirius.ValidationErrors, error) {
```

Comment:
Based on my interface comment above, you'd end up having a new `mockEditMyDetailsClient` with this on it